activerecord/test/cases/relation/field_ordered_values_test.rb

```diff
@@ -109,4 +109,12 @@ def test_in_order_of_with_associations
     books = Book.joins(:author).in_order_of(:"authors.name", order)
     assert_equal(order, books.map { |book| book.author.name })
   end
+
+  def test_in_order_of_with_filter_false
+    order = [3, 4, 1]
+    posts = Post.in_order_of(:id, order, filter: false)
+
+    assert_equal(order, posts.limit(3).map(&:id))
+    assert_equal(11, posts.count)
```

Comment:
WHat's the order for the remaining elements?

Reply:
It's an excellent question. As I see it, arbitrarily, cause the remaining elements will be set with `ELSE` value in `CASE` clause. 

Honestly, i don't know how postgres defines behind the scenes the order for all the equal values setted with else value(if exists something like that).

Reply:
MySQL 8 is always returning the same order:
```ruby
irb(main):001> User.order(Arel.sql("(CASE WHEN id = 8232465 THEN 1 WHEN id = 8232463 THEN 2 ELSE 3 END)")).limit(5).pluck(:id)
  User Pluck (2.8ms)  SELECT `users`.`id` FROM `users` ORDER BY (CASE WHEN id = 8232465 THEN 1 WHEN id = 8232463 THEN 2 ELSE 3 END) LIMIT 5
=> [8232465, 8232463, 8232464, 8232466, 8232468]
```
But I'd not rely in the order of the remaining elements in this scenario. :grimacing:

Reply:
@willianveiga Yeah, definitely.